.github/workflows/action.yml

```diff
@@ -0,0 +1,47 @@
+name: Github action to create EKS cluster using eksctl and install DOP on it
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install eksctl
+      run: |
+        curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+        sudo mv /tmp/eksctl /usr/local/bin
+        sleep 10
+        eksctl version
+
+    - name: Checkout code
+      uses: actions/checkout@master
+
+    - name: Run Shellcheck
+      uses: bewuethr/shellcheck-action@v2
+
+##     TODO:: There is no go-file in repo. Thatswhy we are
+##     commenting this lines for now. 
+#
+#     - name: Run golangci-lint
+#       uses: actions-contrib/golangci-lint@v1
+
+    - name: Configure AWS credentials from secrets
```

Comment:
What happens if there is any error after successful installation of EKS?
Does GitHub action delete the cluster?

Reply:
No, it will not delete I will push one more commit where it will delete cluster if anything fails @AmitKumarDas

Reply:
I have pushed commit for this now if any failure happens then also clean step will get executed @AmitKumarDas